fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingByIndex.java

```diff
@@ -66,12 +68,36 @@
 @API(API.Status.INTERNAL)
 public class IndexingByIndex extends IndexingBase {
     @Nonnull private static final Logger LOGGER = LoggerFactory.getLogger(IndexingByIndex.class);
+    @Nonnull private static final byte[] START_BYTES = new byte[]{0x00};
+    @Nonnull private static final byte[] END_BYTES = new byte[]{(byte)0xff};
+
     @Nonnull private final OnlineIndexer.IndexFromIndexPolicy policy;
+    @Nonnull private final byte[] myTypeStamp;
 
     IndexingByIndex(@Nonnull IndexingCommon common,
                     @Nonnull OnlineIndexer.IndexFromIndexPolicy policy) {
         super(common);
         this.policy = policy;
+        this.myTypeStamp = compileTypeStamp();
+    }
+
+    @Override
+    @Nonnull
+    byte[] getTypeStamp() {
+        return myTypeStamp;
+    }
+
+    @Override
+    boolean matchingTypeStamp(final byte[] stamp) {
+        return Arrays.equals(stamp, myTypeStamp);
+    }
+
+    @Nonnull
+    private byte[] compileTypeStamp() {
```

Comment:
I think this is `byte[]` because you manually serialize/deserialize it. Since you've now changed to protobuf, it's nicer to return the protobuf object, so that you can get the fields easily.

Reply:
Did you fix this?

Reply:
> `private static IndexMetaDataProto.IndexMetaDataIndexingStamp compileIndexingTypeStamp(Index srcIndex)`. 
??